### PR TITLE
Bump librsvg to 2.39.0

### PR DESCRIPTION
**Pull request by alarcher**
Do not merge for now.
2.40.17 is ABI compatible but dropped gtk2 support.

### CODE REVIEW

**Comment by alarcher** [comment]
Been using this for 3 weeks now.

File: components/library/librsvg/Makefile [hunk 1]
```diff
@@ -13,24 +13,30 @@
...
-include ../../make-rules/configure.mk
-include ../../make-rules/ips.mk
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
 
-PATH=/usr/gnu/bin:/usr/bin
+PATH=$(PATH.gnu)
+
+CPPFLAGS+= -P
```
Comment (pyhalov) [inline components/library/librsvg/Makefile]: How is it used?

Comment (alarcher) [reply components/library/librsvg/Makefile]: If I recall the build fails with gcc 7.1 due to line numbers during file generation.

Comment (pyhalov) [reply components/library/librsvg/Makefile]: But where CPPFLAGS are passed to configure?

Comment (alarcher) [reply components/library/librsvg/Makefile]: Good question... I need to check again in the gcc 7 zone if there was a problem. Another question is why are they not in general?

Comment (pyhalov) [reply components/library/librsvg/Makefile]: As all update of build logic, which could affect many components, adding it to default list should be tested. (How will current components behave when they will get CPPFLAGS="" in configure env / parameters?)

Comment (alarcher) [reply components/library/librsvg/Makefile]: I am asking if there a reason why this was not in the original recipe

Comment (pyhalov) [reply components/library/librsvg/Makefile]: don't think there was exact reason

Comment (alarcher) [reply components/library/librsvg/Makefile]: Also originally this recipe was for 2.40.17, could be that it was needed for this version.
In any case I will remove.

**Comment by alarcher** [comment]
Updated.

**Pull request merged**